### DISCUSSION TITLE
Audiogridder Server 1.2.0 stuck on "Loading Plugins Cache"

### DISCUSSION THREAD

**Discussion by ilkraken** [apohl79/audiogridder#1173] [Help] [answered]
[AudioGridderServer_2023-03-09_18-51-20.log](https://github.com/apohl79/audiogridder/files/10934680/AudioGridderServer_2023-03-09_18-51-20.log)
[AudioGridderPlugin_2023-03-09_18-45-00.log](https://github.com/apohl79/audiogridder/files/10934684/AudioGridderPlugin_2023-03-09_18-45-00.log)

Hi guys, I just updated from 1.1.1 and the server does not scan the plugins and shows that message, I can provide only those two logs for now. Any Ideas? Already tried to wipe all and do a clean install of 1.2.0 but still hangs on loading. Maybe I overlooked some folders to erase? I'm on OSX Mojave latest update.

I also tried to update the server on my pc with windows and also the new 1.2.0 server get stuck and gives me the same "loading plugins cache" message.

Thanks in advance!

**Comment by apohl79** [comment, marked as answer]
Looks like you have to re-scan all plugins. Open a terminal and run `rm -rf ~/.audiogridder/audiogridderserver*`.

**Reply by ilkraken** [reply]
Wow that was uber fast and accurate, thank you that fixed it for OSX! Same command but on cmd to fix also the Windows 10 pc identical server issue?

**Reply by apohl79** [reply]
On win10 you better use the explorer and navigate to `C:\Users\<User>\AppData\Roaming\AudioGridder\` and remove all files starting with `audiogridderserver`.

**Reply by ilkraken** [reply]
Beer sent, thank you!

**Reply by apohl79** [reply]
cheers! :-)

**Reply by hydrazoan** [reply]
Just hit this problem myself. Worked like a dream :)
